aten/src/ATen/native/mkldnn/xpu/detail/Attr.h

```diff
@@ -0,0 +1,375 @@
+#pragma once
+
+#include <ATen/ATen.h>
+#include <oneapi/dnnl/dnnl.hpp>
+#include <oneapi/dnnl/dnnl_types.h>
+#include <ATen/native/mkldnn/xpu/detail/Utils.h>
+
+namespace at {
+namespace native::xpu {
+namespace onednn{
```

Comment:
why not `namespace at::native::xpu::onednn`

Reply:
Has changed to `at::native::onednn`, thanks for suggestions.